app/uk/gov/hmrc/rasapi/controllers/FileController.scala

```diff
@@ -93,7 +92,7 @@ class FileController @Inject()(
             (parseStringIdToBSONObjectId(fileId) match {
               case Success(bsonId) => chunksRepo.removeChunk(bsonId).map { isChunkRemoved =>
                 if (isChunkRemoved) {
-                  Logger.warn(s"[FileController][remove] Chunk deletion succeeded, fileId is: ${fileId}")
+                  Logger.info(s"[FileController][remove] Chunk deletion succeeded, fileId is: ${fileId}")
```

Comment:
Not changed as part of this PR but line 105, the error log around chunk deletion doesn't need to be at error either! An error log gets thrown in the removeChunk function already.